.github/workflows/test-go.yaml

```diff
@@ -69,14 +66,14 @@ jobs:
 
     - name: Run Go Tests
       run: |
-        GO_TEST_EXTRA_FLAGS="-v -json -race=$RACE_ENABLED -timeout=$GO_TEST_TIMEOUT" \
+        GO_TEST_EXTRA_FLAGS="-v -race=$RACE_ENABLED -timeout=$GO_TEST_TIMEOUT" \
           TEST_LOCK_FILE_PATH=$(pwd)/lock \
           NETWORK_TEST=1 \
           REDIS_TEST=1 \
           MYSQL_TEST=1 \
           MINIO_STORAGE_TEST=1 \
           SAML_IDP_TEST=1 \
-          make test-go 2>&1 | tee /tmp/gotest.log | gotestfmt
+          make test-go 2>&1 | tee /tmp/gotest.log
```

Comment:
So I kept the `tee` part as we upload `gotest.log` at the last step of this workflow, not sure what this is for? Would it be affected by not being a JSON output anymore?